src/bar-chart-trellis/BarChartTrellis.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { FacetController, OrdinalFrame } from "semiotic";
+import styled from "styled-components";
+import { SENTENCE_LENGTH_KEYS, SENTENCE_LENGTHS, THEME } from "../constants";
+import ChartWrapper from "../chart-wrapper";
+import Tooltip from "../tooltip";
+
+const CHART_HEIGHT = 360;
+const CHART_MIN_WIDTH = 320;
+
+const MARGIN = { top: 40, bottom: 56, left: 48, right: 0 };
+
+const Wrapper = styled(ChartWrapper)`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const ChartTitle = styled.text`
+  text-anchor: start;
+`;
+
+const ColumnLabel = styled.text`
+  text-anchor: middle;
+`;
+
+const renderTooltip = (chartTitle) => ({ pieces: [d] }) => (
+  <Tooltip>
+    {chartTitle}
+    <br />
+    {d.value} serving {d.label} year(s)
+  </Tooltip>
+);
+
+export default function BarChartTrellis({ data, width }) {
+  const [highlightedLabel, setHighlightedLabel] = useState();
+
+  let chartWidth = data.length > 1 ? width / 2 : width;
+
+  let alternatingAxes = true;
+  if (chartWidth < CHART_MIN_WIDTH) {
+    chartWidth = width;
+    alternatingAxes = false;
+  }
+
+  const axes = [
+    { baseline: false, orient: "left", tickLineGenerator: () => null },
+  ];
+
+  return (
+    <Wrapper>
+      <FacetController
+        baseMarkProps={{
+          transitionDuration: { fill: THEME.transition.defaultDurationMs },
+        }}
+        margin={MARGIN}
+        oAccessor="label"
+        oLabel={(label) => {
+          const postfix =
+            label === SENTENCE_LENGTHS.get(SENTENCE_LENGTH_KEYS.lessThanOne)
+              ? " year"
+              : "";
+          return <ColumnLabel>{`${label}${postfix}`}</ColumnLabel>;
+        }}
+        oPadding={8}
+        rAccessor="value"
+        sharedRExtent
+        size={[chartWidth, CHART_HEIGHT]}
+        style={(d) => ({
+          fill: highlightedLabel === d.label ? THEME.colors.highlight : d.color,
+        })}
+        type="bar"
+      >
+        {data.map(({ title, data: chartData }, index) => (
+          <OrdinalFrame
+            axes={alternatingAxes && index % 2 ? undefined : axes}
+            customHoverBehavior={(d) =>
+              d ? setHighlightedLabel(d.column.name) : setHighlightedLabel()
```

Comment:
It's not clear if this is intentional or not, but when I select a dimension and then hover over a sentence length, the bars for each category within that dimension of the same sentence length are also highlighted (as opposed to just the one I'm hovering over).  See the screenshot below:

<img width="1052" alt="Screen Shot 2020-07-24 at 7 12 35 AM" src="https://user-images.githubusercontent.com/25503/88387858-66f48e80-cd81-11ea-9170-19731394b00b.png">

Reply:
yeah, intentional but not specified; I ran it by juan and he is happy with it